src/public/app/core/types/ScPlayer.ts

```diff
@@ -9,20 +9,20 @@ export interface ScPlayer {
 }
 
 export enum ScPlayerEvent {
-    StateChange = 'state-change',
-    Play = 'play',
-    PlayStart = 'play-start',
-    PlayResume = 'play-resume',
-    Pause = 'pause',
-    Finished = 'finish',
-    Seek = 'seek',
-    Seeked = 'seeked',
-    GeoBlocked = 'geo_blocked',
-    BufferingStart = 'buffering_start',
-    BufferingEnd = 'buffering_end',
-    AudioError = 'audio_error',
-    Time = 'time',
-    NoStreams = 'no_streams',
-    NoProtocol = 'no_protocol',
-    NoConnection = 'no_connection'
+    StateChange = <any>'state-change',
+    Play = <any>'play',
+    PlayStart = <any>'play-start',
+    PlayResume = <any>'play-resume',
+    Pause = <any>'pause',
```

Comment:
Were going to end up using all of them

On Tue, Apr 19, 2016 at 5:53 PM, dswen <notifications@github.com> wrote:

> In src/public/app/core/types/ScPlayer.ts
> <https://github.com/ryanquinn3/beetz/pull/8#discussion_r60336754>:
>
> > -    Finished = 'finish',
> > -    Seek = 'seek',
> > -    Seeked = 'seeked',
> > -    GeoBlocked = 'geo_blocked',
> > -    BufferingStart = 'buffering_start',
> > -    BufferingEnd = 'buffering_end',
> > -    AudioError = 'audio_error',
> > -    Time = 'time',
> > -    NoStreams = 'no_streams',
> > -    NoProtocol = 'no_protocol',
> > -    NoConnection = 'no_connection'
> > +    StateChange = <any>'state-change',
> > +    Play = <any>'play',
> > +    PlayStart = <any>'play-start',
> > +    PlayResume = <any>'play-resume',
> > +    Pause = <any>'pause',
>
> also let's avoid writing big definitions like this when we're only gonna
> use a few. We can add extras as we go, but it adds more difficulty in
> reading and refactoring if we have unused ones.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ryanquinn3/beetz/pull/8/files/2aabd0b9fad287cfc337421e70eb4c4f70cfdaed#r60336754>
>